control/lti.py

```diff
@@ -417,12 +465,17 @@ def evalfr(sys, x):
 
     .. todo:: Add example with MIMO system
     """
-    if issiso(sys):
-        return sys.horner(x)[0][0]
-    return sys.horner(x)
+    out = sys.horner(x)
+    if not hasattr(x, '__len__'): 
+            # received a scalar x, squeeze down the array along last dim
+            out = np.squeeze(out, axis=2)
+    if squeeze and issiso(sys):
```

Comment:
squeeze parameter not used as advertised

Reply:
There are two squeezes here. This One insures that if we received a scalar s, the output matches: scalar for SISO, 2d matrix for mimo. Will add a comment explaining.

Reply:
I find the parameter can too easily be confused with the behavior of the  `squeeze` parameter from the `timeresp` functions. and the one in `iosys'. Maybe rename?

Reply:
posted a new issue about this #453 with a proposal to resolve this discrepancy in how frequency response and time responses are squeezed depending on mimo or silo.

Reply:
I will leave this as it is until we get to a conclusion on #453.